app/client/views/login/login.html

```diff
@@ -1,5 +1,6 @@
 <div id="particles-js"></div>
```

Comment:
`#particles-js` is still first in the hierarchy, it needs to be moved after `#login`